scss/_utilities.scss

```diff
@@ -613,6 +629,11 @@ $utilities: map-merge(
         100: 1
       )
     ),
+    "subtle-background-color": (
```

Comment:
Don't we need the following for this utility as well?

```
local-vars: (
    "bg-opacity": 1
),
```

Otherwise `.bg-body-secondary` and `.bg-primary-subtle` (for example) won't have exactly the same behavior.

If they can have the same behavior, you could use `map-merge-multiple` and do:

```
"background-color": (
      property: background-color,
      class: bg,
      local-vars: (
        "bg-opacity": 1
      ),
      values: map-merge-multiple(
        $utilities-bg-colors,
        $utilities-bg-subtle,
        (
          "transparent": transparent,
          "body-secondary": rgba(var(--#{$prefix}secondary-bg-rgb), var(--#{$prefix}bg-opacity)),
          "body-tertiary": rgba(var(--#{$prefix}tertiary-bg-rgb), var(--#{$prefix}bg-opacity)),
          "body-emphasis": rgba(var(--#{$prefix}emphasis-bg-rgb), var(--#{$prefix}bg-opacity)),
        )
      )
    ),
```

and so remove these 5 lines

⚠️ This comment is linked to the other additions in this file so could be adapted to the newly added `"subtle-border-color"` and `"text-color"` that don't have:

```
local-vars: (
    "border-opacity": 1
),
```

Reply:
Great idea!

Reply:
Actually this caused some other issues. Reverted for now.

Right now none of the new colors have `-rgb` variations. By design, I didn't include those values in addition to the hex values because it'd be soooooo many new CSS vars on the `:root`. I'd need to refactor more things here if we wanted to support the CSS variables on all these.